### PR TITLE
LEGO and edits to spin

### PR DESCRIPTION
**Pull request by thomaswrona**
I assumed that the spin entry was the one that already exists.

### CODE REVIEW

**Comment by thomaswrona** [comment]
Not quite done with round 2 edits, and may add more formulas to the spin code

File: codes/quantum/spins/spins_into_spins.yml [hunk 1]
```diff
@@ -8,12 +8,33 @@ physical: spins
 logical: spins
 
 name: 'Spin code'
```
Comment (valbert4) [inline codes/quantum/spins/spins_into_spins.yml:10]: This is an umbrella entry for the Gross codes but also others. Please create a new entry for the J. Gross codes on next round.

Comment (thomaswrona) [reply codes/quantum/spins/spins_into_spins.yml:10]: Does J. Gross codes refer to the example in "Encoding a qubit in a spin"

**Comment by valbert4** [comment]
Yep

Best,
Victor

File: codes/quantum/spins/spins_into_spins.yml (hunk 1 above)
Comment (valbert4) [reply codes/quantum/spins/spins_into_spins.yml:10]: Yes

Comment (valbert4) [reply codes/quantum/spins/spins_into_spins.yml:10]: Please resubmit with the text you added here moved to a new entry for "Gross spin code" and you'll be all set. Thanks.

File: codes/quantum/properties/lego.yml [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+    - 'We then look at the codes graphically, treating each \(i_j\) as an edge dangling out of the tensor vertex \(V_{i_1 \ldots i_{n+k}}\). These edges are either connected to another tensor vertex''s edges or left dangling.'
+    - 'Each local tensor has unitary product stabilizers (UPS). The goal is to push each UPS through the tensor network until each dangling edge has only trivial support. Otherwise, a matching value is pushed through the edge and the process is repeated on the next tensor. If a UPS can be pushed through the whole network, then a UPS for the larger network has been found.'
+    - 'The dangling legs (edges) and UPS of the whole network can then be converted to physical/logical elements and stabilizers/logical operators for a new quantum ECC.'
+
+  decoders:
+    - 'The decoder is created by creating a decoding quantum circuit with dangling legs replaced with input/output wires, and tensors converted to unitary gates.'
+    - 'Maximum likelihood decoding can be used when the tensors are stabilizer codes.'
+
+notes:
+  - 'The bounds for these codes depend entirely on the codes used to create them.'
```
Comment (valbert4) [inline codes/quantum/properties/lego.yml:29]: Is first bullet really necessary?

File: codes/quantum/properties/lego.yml [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+  - 'These codes can serve as a bridge between quantum error correction and physics. For example, the introduction paper states that finding the tensor constructions of well-known gauge codes using the lego method can help with the understanding of low energy Hamiltonian behavior with frustration in the ground space.'
```
Comment (valbert4) [inline codes/quantum/properties/lego.yml:30]: If there is no quantitative support for how the method "can help" then we can leave this out

File: codes/quantum/properties/lego.yml [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+  - 'Many known codes can be created using this code''s methods in order to further their understanding. Examples can be found in the paper that introduces these codes.'
+  - 'Tensor-network codes are types of LEGO codes where logical and physical legs are pre-assigned and logical legs are not contracted.'
```
Comment (valbert4) [inline codes/quantum/properties/lego.yml:32]: This is interesting; please elaborate. What is a tensor-network code?

File: codes/quantum/spins/spins_into_spins.yml [hunk 5]
```diff
@@ -8,12 +8,33 @@ physical: spins
...
+features:
+
+  encoders:
+    - 'Given any half-integer spin above \(3/2\), explicit codewords can be produced.'
+    - 'Producing codewords begins by creating a projection  onto an irreducible representation of \(2O\).'
+    - 'Then, irreducible representations of the Paulis \(\bar{\sigma}_w\) are defined using the above projection and the angular momentum operators.'
+    - 'Finally, \(|\bar{0} \rangle\) is defined as the \(+1\) eigenvalue of \(\bar{\sigma}_z\) and \(|\bar{1} \rangle = \bar{\sigma}_x |\bar{0} \rangle \).'
+
+  general_gates:
+    - 'Universal computation results from being able to prepare a single logical state, perform one measurement, the phase gate, the Hadamard gate, the conditional phase gate, and the square root of the phase gate.'
```
Comment (valbert4) [inline codes/quantum/spins/spins_into_spins.yml:32]: Incomplete here. How do the 2O irreps allow for all of these things? I.e., How are these gates performed?

**Pull request closed without merging**